src/services/user.service.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-useless-catch */
```

Comment:
is there a reason we are disabling eslint here?

Reply:
It was reporting that I shouldnt use a try catch method and I honestly didnt know how to deal with that even after making enquiries and Googling. You could have a look at the code and help me out. Thanks

Reply:
@okabamac have checked out this link? https://eslint.org/docs/rules/no-useless-catch